spec/models/services/facebook_spec.rb

```diff
@@ -22,14 +22,46 @@
       post_params = @service.create_post_params(post)
     end
 
-    it 'does not add post link when no photos' do
+    it "adds URL of post to end if no link at end" do
       message = "Some text."
       post = double(message: double(plain_text_without_markdown: message, urls: []), photos: [])
       post_params = @service.create_post_params(post)
-      expect(post_params[:message]).not_to include "http"
+      expect(post_params[:message]).to include "(via http:"
```

Comment:
You should check if the via link is acutally at the end if you write in the test-text "to the end".

Reply:
for each test? ok.

Reply:
No, as mentioned in https://github.com/diaspora/diaspora/pull/7774#discussion_r180253983 you can remove most of the tests, so you con't need to check it in each test.